list_koji_tags.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
I'd not use a file extension so you're free to change the implementation language.